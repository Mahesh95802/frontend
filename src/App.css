.App {
  overflow: hidden;
  font-family: 'Montserrat', sans-serif;
}

.basic-padding{
  padding-left: 30px;
  padding-right: 30px;
}

.rounded-corners{
  border-radius: 10px;
}
