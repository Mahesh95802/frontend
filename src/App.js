import React from 'react';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import LoginPage from './pages/LoginPage';
import RegisterPage from './pages/RegisterPage';
import HomePage from './pages/HomePage';
import ProtectedRoutes from './utils/ProtectedRoutes';
import NotFoundPage from './pages/NotFoundPage';

import './App.css';

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Routes>
          <Route path="/login" element={<LoginPage />} />
          <Route path="/register" element={<RegisterPage />} />
          <Route path="/home" element={<ProtectedRoutes><HomePage /></ProtectedRoutes>} />
          <Route path="*" element={<NotFoundPage />} />
        </Routes>
      </BrowserRouter>
    </div>
  );
}

export default App;
