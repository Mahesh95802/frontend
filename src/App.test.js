import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('axios');

test('renders App correctly', () => {
	const { asFragment } = render(<App />);
	expect(asFragment()).toMatchSnapshot();
});
