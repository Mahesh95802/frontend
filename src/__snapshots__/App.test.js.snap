// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="404"
    >
      <h1>
        404
      </h1>
      <h2>
        NotFound
      </h2>
    </div>
  </div>
</DocumentFragment>
`;
